app/src/main/scala/org/alephium/explorer/api/AddressesEndpoints.scala

```diff
@@ -28,6 +28,8 @@ import org.alephium.protocol.Hash
 // scalastyle:off magic.number
 trait AddressesEndpoints extends BaseEndpoint with QueryParams {
 
+  private val activeAddressesMaxSize: Int = 200
```

Comment:
Nit: maybe 4 * 20 since the gas limit of address discovery is 20 by default?

Reply:
discussed this with front-end team, we will go with what you propose to follow the more closely bip44